File: GateOne/DayFour/java/TicTacToe.java
import java.util.Scanner;

public class TicTacToe{

public static void main(String[] args){

Scanner input = new Scanner(System.in);

System.out.println("              WELCOME TO A GAME OF TIC TAC TOE");

System.out.print("please Register your name: ");
String name = input.nextLine();

System.out.println(name + " This is an example of how the game looks below");

System.out.println("x|o|x");
System.out.println("x|o|x");
System.out.println("o|x|o");

for(int counter = 1; counter < 10; counter++){

	System.out.print("Enter between X or O: ");
	String choice = input.nextLine();
	
	if (choice != "x" && choice != "o"){
		System.out.print("Which column do u want ur input to go: ");
		int column = input.nextInt();
	}

	else{
		System.out.print("Wrong input try again");
	}

}

}

}

File: GateOne/DayFour/python/TicTacToe.py
def print_board(board):
    for row in board:
        print(" ".join(row))

def main():
    print("WELCOME TO A GAME OF TIC TAC TOE")
    name = input("Please register your name: ")
 
    for _ in range(9):
        choice = input("Enter X or O: ")
        row, col = map(int, input("Which row and column do you want your input to go? (e.g., 1 2): ").split())
        board[row - 1][col - 1] = choice
        print_board(board)

